app/views/layouts/application.html.erb

```diff
@@ -29,5 +29,7 @@
   </div>
 </div>
 
+<%= javascript_include_tag "application.js" %>
```

Comment:
I personally am against modules that self-install so would like to move away from that pattern.